docs/application/native/guides/location-sensors/user-awareness.md

```diff
@@ -0,0 +1,155 @@
+    ```
+    The ```__presence_detected_cb()``` function is a callback, which is called when the user presence is detected.
+  
+2.  Using the user presence detection consumes the system resources, so if the service is not used, stop the user presence detection using the ```ua_monitor_stop_presence_detection()``` API. Call the ```ua_monitor_start_presence_detection()``` API again if user presence detection is needed.
+    ```
+    ua_monitor_stop_presence_detection(g_ua_mon_h);
+    ```
+  
+## Start the absence detection
+
+To start the absence detection, follow these steps:
+  
+1.  Start the user absence detection using the `ua_monitor_start_absence_detection()` API. This call is asynchronous and only initiates the process of starting the user absence detection. Once the absence detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user absence is detected, use the `ua_absence_detected_cb()` function:
+    ```
+    ua_monitor_start_absence_detection(g_ua_mon_h, NULL, detection_mode, __absence_detected_cb, NULL);
+    ```
+    The `__absence_detected_cb()` is a callback function, which is called when the user absence is detected. 
+    
+2.  Using the user absence detection consumes the system resources, so if the service is not used, stop the user absence detection using the `ua_monitor_stop_absence_detection()` API. Call the `ua_monitor_start_absence_detection()` API again if user presence detection is needed:
+    ```
+    ua_monitor_stop_absence_detection(g_ua_mon_h);
+    ```
+ 
+  
+## Start the location detection
+
+To start the location detection, follow these steps:
+
+1.  Start the user location detection using the `ua_monitor_start_location_detection()` API. This call is asynchronous and only initiates the process of starting the user location detection. Once the location detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user location is detected, use the `ua_location_detected_cb()` function:
+    ```
+    ua_monitor_start_location_detection(g_ua_mon_h, NULL, detection_mode, __location_detected_cb, NULL);
+    ```
+    The `__location_detected_cb()` is a callback function, which is called when the user location is detected.
+    
+2.  Using the user location detection consumes the system resources, so if the service is not used, stop the `ua_monitor_stop_location_detection()` API. Call the `ua_monitor_start_location_detection()` API again if user location detection is needed: 
+    ```
+    ua_monitor_stop_location_detection(g_ua_mon_h);
+    ```
+  
+## Release all resources
+At the end of application, destroy all used resources (such as, user handle, service handle, device handle, etc.) as below:
+    
+    ```
+    ua_device_destroy(g_device_h);
+    ua_user_destroy(g_user_h);
+    ua_service_destroy(g_service_h);
+    ua_monitor_destroy(g_ua_mon_h);
+    ```
```

Comment:
Hello, @maliha-srbd It is because of the markdown syntax.

**\`\`\`** is used to open & close the code block.
However **\`\`\`** is also used in suggestion box.

Your suggestion would be correct, that it would be in form like below.

\`\`\`suggestion&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//1st \`\`\`
\`\`\`&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//2nd \`\`\`
ua_device_destroy(g_device_h);
ua_user_destroy(g_user_h);
ua_service_destroy(g_service_h);
ua_monitor_destroy(g_ua_mon_h);
\`\`\`&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//3rd \`\`\`
\`\`\`&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//4th \`\`\`

Unfortunately, so the github thinks the 1st and 2nd \`\`\` are making a code block,
and the 3rd and 4th \`\`\` are making another code block. As the result, the suggestion box can't be generated as the intention.
Please explain in words in this case, and check the updated result.

Reply:
Just one more point to check.
The spaces in front of the code lines should be removed also.


\`\`\`
ua_device_destroy(g_device_h);
ua_user_destroy(g_user_h);
ua_service_destroy(g_service_h);
ua_monitor_destroy(g_ua_mon_h);
\`\`\`

Reply:
Hi @anujk-singh , hope you're well. 
The issue regarding the **lines# 145-150** need changes from your end. Please remove the spaces that exist before the mentioned lines. There should be no space before both the code block tags as well as the codes.
For some reason if we suggest these changes from our end, the format seems to break. 
So could you please make the suggestions from your end and remove the spaces?? It should then look like this:

```
ua_device_destroy(g_device_h);
ua_user_destroy(g_user_h);
ua_service_destroy(g_service_h);
ua_monitor_destroy(g_ua_mon_h);
```

Please let me know if you have any confusion regarding this.